### PR TITLE
Fix decimal for swap output

### LINKED ISSUE #1173: [Swap] Fix decimal for output result
For example: Test asset `ETH.USDT` (Note: That's not the original `ETH.USDT`) should have 6 decimal only

Screenshot from 2021-03-24 14-37-44

### PR DESCRIPTION
**Pull request by veado**
- [x] Swap: Fix decimal for output 
- [x] Remove unneeded fee error handling for target chain
- [x] Rename few `xyzAmount` values / functions to `xyzAmountMax1e2`
- [x] Enable slider for unlocked wallet user only
- [x] Use `getTwoSigfigAssetAmount` in `formatFee` helper 

Fix #1173

### CODE REVIEW

**Review by sarawut11 — APPROVED** [review]
Cool 👍

File: src/renderer/views/swap/SwapView.tsx [hunk 1]
```diff
@@ -96,10 +96,21 @@ export const SwapView: React.FC<Props> = (_): JSX.Element => {
...
+        O.fold(
+          () => Rx.of(RD.failure(Error(intl.formatMessage({ id: 'swap.errors.asset.missingTargetAsset' })))),
+          (asset) => assetWithDecimal$(asset, network)
+        )
+      ),
+    [assetWithDecimal$, intl, network, oRouteTarget]
   )
 
+  const targetAssetRD: AssetWithDecimalRD = useObservableState(targetAssetDecimal$, RD.initial)
+  // const targetAssetRD: AssetWithDecimalRD = RD.success({ asset, decimal: 18 })
```
Comment (junkai121) [inline src/renderer/views/swap/SwapView.tsx:112]: Do we need this line anymore?

**Review by junkai121 — APPROVED** [review]
LGTM!

File: src/renderer/views/swap/SwapView.tsx (hunk 1 above)
Comment (veado) [reply src/renderer/views/swap/SwapView.tsx:112]: No, I will remove it...

File: src/renderer/components/swap/Swap.tsx [hunk 2]
```diff
@@ -125,7 +124,11 @@ export const Swap = ({
 }: SwapProps) => {
   const intl = useIntl()
 
+  const unlockedWallet = useMemo(() => isLocked(keystore) || !hasImportedKeystore(keystore), [keystore])
```
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/swap/Swap.tsx:127]: Ok! _I will change it with another PR, we do need it for testing now..._

**Pull request merged**